index.d.ts

```diff
@@ -0,0 +1,12 @@
+import {CLIEngine} from 'eslint';
+
+/**
+Format the lint results.
+
+@param results - Lint result for the individual files.
+@param data - Extended information related to the analysis results.
+@returns The formatted output.
+*/
```

Comment:
I think the doc comment should be added to the type in DefinitelyTyped. That way every consumer benefits from it and it can be properly kept in sync with ESLint changes.

Reply:
Good idea. Please, go ahead and create PR.

Reply:
I'm sorry, but I don't have time to do that. You are the one that wants this to be merged.